MVP Module/Tests/Feed UI/Helpers/FeedViewController+TestHelpers.swift

```diff
@@ -44,6 +44,18 @@ extension FeedViewController {
 		return refreshControl?.isRefreshing == true
 	}
 
+	var isShowingLoadingErrorView: Bool {
+		return tableView.tableHeaderView?.alpha == 1
+	}
+
+	var loadingErrorMessage: String {
+		(tableView.tableHeaderView as? ErrorView).map { $0.message ?? "" } ?? ""
+	}
```

Comment:
Replacing a `nil` message with empty string `""` adds behavior to the test. 

The test shouldn't replace any values you receive from the production code. Otherwise, the test is not checking the actual prod behavior. 

Here, it should return optional instead:

```swift
	var loadingErrorMessage: String? {
		errorView.message
	}
```

Moreover, you can remove the 'loading' from the name since you should decouple the concepts of loading and errors:

```suggestion
	var errorMessage: String? {
		errorView.message
	}
```